### ISSUE TITLE
on gqlgen get unable to parse config: yaml: unmarshal errors

### ISSUE THREAD

**Issue by bjm88** [99designs/gqlgen#825]
### What happened?
go run github.com/99designs/gqlgen -v
unable to parse config: yaml: unmarshal errors:
  line 18: key "deprecated" already set in map
  line 20: key "include" already set in map
  line 22: key "skip" already set in map

### What did you expect?
no errors on regenerate

### Minimal graphql.schema and models to reproduce
type Communication {
  identifier: [Identifier!]
  status: String!
  category: [CodeableConcept!]
  priority: String!
  medium: [CodeableConcept!]
  subject: Patient!
  topic: CodeableConcept!
  sent: Time!
  received: Time!
  recipient: [Patient!]
  sender: Organization!
  reasonCode: [CodeableConcept!]
  payload: [Message!]
  note: [Annotation!]
}

type ServiceRequest {
  identifier: [Identifier!]
  status: String!
  intent: String!
  category: [CodeableConcept!]
  priority: String!
  code: [CodeableConcept!]
  orderDetail: [CodeableConcept!]
  subject: Patient!
  encounter: Encounter!
  occurrenceDateTime: Time!
  performer: [Practitioner!]
  locationCode: [CodeableConcept!]
  locationReference: [Location!]
  reasonCode: [CodeableConcept!]
  insurance: [Coverage!]
  supportingInfo: [Message!]
  note: [Annotation!]
  patientInstruction: String
}

type Annotation {
  author: [Practitioner!]
  time: Time
  text: String!
}

type Coverage {
  identifier: [Identifier!]
  status: String!
  type: [CodeableConcept!]
  policyHolder: Patient
  subscriberId: String
  payor: [Patient!]
  order: Int
  network: String
}

type Location {
  identifier: [Identifier!]
  name: String
  telecom: [ContactPoint!]
  address: [Address!]
  physicalType: [CodeableConcept!]
  hoursOfOperation: [HoursOfOperation!]
}

type HoursOfOperation {
  daysOfWeek: String
  allDay: Boolean
  openingTime: Time
  closingTime: Time
}

type ContactPoint {
  system: String
  value: [String!]
  use: [String]
}

type Practitioner {
  identifier: [Identifier!]
  active: Boolean
  name: [HumanName!]
  address: [Address!]
  gender: String
  birthDate: Time
}

type Encounter {
  identifier: [Identifier!]
  status: CodeableConcept!
  type: [CodeableConcept!]
  serviceType: [CodeableConcept!]
  priority: [CodeableConcept!]
  subject: Patient!
  serviceProvider: [Organization!]
}

type Message {
  contentString: String!
}

type CodeableConcept {
  coding: Coding!
  text: String!
}

type Coding {
  system: String!
  version: String!
  code: String!
  display: String!
  userSelected: Boolean!
}

type Organization {
  identifier: [Identifier!]
  type: [CodeableConcept]
  name: String!
  endpoint: [Endpoint!]
}

type Patient {
  identifier: [Identifier!]
  name: [HumanName]
  gender: String!
  birthDate: Time!
  address: [Address!]
}

type Identifier {
  use: String!
  type: CodeableConcept!
  system: String!
  value: String!
}

type HumanName {
  family: String!
  given: [String!]
}

type Address {
  use: String!
  type: String!
  line: [String!]
  city: String!
  district: String!
  state: String!
  postalCode: String!
  country: String!
}

type Endpoint {
  address: String!
}

input CommunicationInput {
  medium: String!
  payload: String!
  patientID: String!
  orgID: String!
}

type Query {
  communication(id: ID!): Communication
  communications: [Communication!]!
}

type Mutation {
  createCommunication(communication: CommunicationInput!): Communication!
    @hasPermission(permission: "send_patient_communication")
  updateCommunication(
    id: ID!
    communication: CommunicationInput!
  ): Communication
}

schema {
  query: Query
  mutation: Mutation
}

scalar Time

directive @hasPermission(permission: String!) on FIELD_DEFINITION

### versions
 - `gqlgen version`?  v0.9.1

 - `go version`? go version go1.12.1 darwin/amd64

 - dep or go modules? go mod

**Comment by Schparky** [comment]
You can remove (or comment) the following lines from `gqlgen.yml` as follows:
```yml
#directives:
#  deprecated:
#    skip_runtime: true
#  include:
#    skip_runtime: true
#  skip:
#    skip_runtime: true
```

These options have been included by default.

**Comment by vektah** [comment]
These are some internal options that were accidentally exposed in the generated config in 0.9.1. It was fixed in 0.9.2, removing those options from the config will get you going on either version.
